gammapy/cube/tests/test_fit.py

```diff
@@ -99,20 +92,32 @@ def counts(sky_model, exposure, background, psf, edisp):
 
 @requires_dependency("iminuit")
 @requires_data("gammapy-extra")
-def test_cube_fit(sky_model, counts, exposure, psf, background, mask, edisp):
+def test_cube_fit(sky_model):
+    ebounds = np.logspace(-1.0, 1.0, 3)
+    ebounds_true = np.logspace(-1.0, 1.0, 4)
+    geom_r = geom(ebounds)
+    geom_t = geom_etrue(ebounds_true)
+
+    background_map = background(geom_r)
+    psf_map = psf(geom_t)
+    edisp_map = edisp(geom_r, geom_t)
+    exposure_map = exposure(geom_t)
+    counts_map = counts(sky_model, exposure_map, background_map, psf_map, edisp_map)
+    mask_map = mask(geom_r, sky_model)
+
     sky_model.parameters["lon_0"].value = 0.5
```

Comment:
Could you move this code to the `sky_model` fixture?

Reply:
Moving this to the `sky_model()` will also modify the values of the initial simulation.
That spoils things, right?

Reply:
You are right of course. Alternatively you could define a second fixture `sky_model_fit`, that is used for the fitting....

Reply:
Ok! Changes made.